driver-scala/src/main/scala/org/mongodb/scala/model/changestream/FullDocumentBeforeChange.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mongodb.scala.model.changestream
+
+import com.mongodb.client.model.changestream.{ FullDocumentBeforeChange => JFullDocumentBeforeChange }
+
+import scala.util.Try
+
+/**
+ * Change Stream fullDocumentBeforeChange configuration.
+ *
+ * Determines what to return for update operations when using a Change Stream. Defaults to [[FullDocumentBeforeChange#DEFAULT]].
+ *
+ * @note Requires MongoDB 6.0 or greater
+ * @since 4.7
+ */
+object FullDocumentBeforeChange {
```

Comment:
The documentation for values here differs from that of values of `FullDocumentBeforeChange` in the Java API (the one in the Java API appears to be the correct one).

Reply:
Fixed